ratelimit/config/ratelimit/v3/rls_conf.pb.go

```diff
@@ -0,0 +1,551 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
Some questions on this generated pb file...

Where does this come from? How can we automate that it keeps up with changes from its source? If this is from the envoy data-plane mirror or the udpa repo, why is it receiving it's own folder instead of living under the `envoy/` generated code folder.

Can you give me some background here?

Reply:
I planned to keep the source proto file in the rate limit service repository https://github.com/envoyproxy/ratelimit/pull/371 since it contains the rate limit config message.

The pb file is generated on my local machine (definitely we have to automate it 🙂). The location to keep this pb file is just what I decided, we can move it to a proper location. Could you please help me on automating this pb file generation?

It is better if we can keep the source proto file in the rate limit repository, but I think it is not an issue to keep that in another repository.

Reply:
Okay no worries let's definitely keep the proto file close to the implementation. 

I think it's worth moving it. I'm about to travel for the day so I will think about the automation and where we want to keep the generated code.